x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/detection_rules_client/methods/create_prebuilt_rule.ts

```diff
@@ -31,5 +33,5 @@ export const createPrebuiltRule = async (
     data: internalRule,
   });
 
-  return rule;
+  return transformValidate(rule);
```

Comment:
Shouldn't we try-catch this one as well?

Reply:
Thanks! Updated in [`3d37bd6`](https://github.com/elastic/kibana/pull/185748/commits/3d37bd6c45b31abe06685280546a4d93997d64de).